packages/feature-flags/src/get-static-flags.js

```diff
@@ -0,0 +1,22 @@
+import { promises as fsPromises } from 'fs';
+import { fileURLToPath } from 'url';
+import path, { dirname } from 'path';
+import pino from '@pins/applications.web/src/server/lib/logger.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+export default async () => {
+	try {
+		const filePath = path.join(__dirname, 'static-feature-flags.json');
+		const data = await fsPromises.readFile(filePath, 'utf8');
```

Comment:
For this you could use `fs.readFileSync` and that way get rid of the `async/await`.

Reply:
cool - done